allennlp/models/biaffine_dependency_parser.py

```diff
@@ -83,15 +93,33 @@ def __init__(self,
         self.encoder = encoder
 
         encoder_dim = encoder.get_output_dim()
-        self.head_arc_projection = torch.nn.Linear(encoder_dim, arc_representation_dim)
-        self.child_arc_projection = torch.nn.Linear(encoder_dim, arc_representation_dim)
+
+        if arc_feedforward is not None:
+            self.head_arc_feedforward = arc_feedforward
+            self.child_arc_feedforward = copy.deepcopy(arc_feedforward)
+        else:
+            self.head_arc_feedforward = FeedForward(encoder_dim, 1,
+                                                    arc_representation_dim,
+                                                    Activation.by_name("elu")())
+            self.child_arc_feedforward = FeedForward(encoder_dim, 1,
+                                                     arc_representation_dim,
+                                                     Activation.by_name("elu")())
```

Comment:
nit: it would be slightly cleaner / simpler if you did

```
self.head_arc_feedforward = head_arc_feedforward or FeedForward(...)
self.child_arc_feedforward = copy.deepcopy(self.head_arc_feedforward)
```

which I think amounts to the same thing?